docs/spec/protocol-modules/storage/assignment.md

```diff
@@ -53,50 +52,52 @@ The `AssignmentCoordinator` can be used to get the `EncodingParams` struct in th
 
 The standard assignment coordinator implements a very simple logic for determining the number of chunks per node and the chunk length, which we describe here. More background concerning this design can be found in the [Design Document](../../../design/assignment.md)
 
-**Index Assignment**.
-For each operator $i$, let $S_i$ signify the amount of stake held by that operator. The number of chunks assigned to an operator $i$ is given by the equation
 
-$$m_i = \text{ceil}\left(\frac{\rho nS_i}{\sum_j S_j}\right) \tag{1}$$
+**Chunk Length**.
 
-where $n$ is the total number of operators and $\rho$ is called the quantization factor.
+The protocol requires that chunk lengths are sufficiently small that operators with a small proportion of stake are able to receive a quantity of data commensurate with their stake share. For each operator $i$, let $S_i$ signify the amount of stake held by that operator. 
 
-**Chunk Length**.
-To determine the chunk length, we first set the reconstruction threshold at the level of chunks (i.e., the number of chunks from which we wish to be able to reconstruct the original blob) as
+We require that the chunk size $C$ satisfy
 
-$$m = ceil(n\rho\gamma)$$
+$$
+C \le \text{NextPowerOf2}\left(\frac{B}{\gamma}\max\left(\frac{\min_jS_j}{\sum_jS_j}, \alpha \right) \right)
+$$
 
-where  $\gamma = \beta-\alpha$, with $\alpha$ and $\beta$ as defined in the [Storage Overview](./overview.md).
 
-We can then derive the chunk length as $C \ge B/m$, where $B$ is the length of the blob.
+where $\gamma = \beta-\alpha$, with $\alpha$ and $\beta$ as defined in the [Storage Overview](./overview.md) and $\alpha = 1/8192$ is a system parameter.
```

Comment:
Since alpha is set to 1/8192, so its not the same definition as in Storage Overview (where alpha is adversial threshold)?

Reply:
Good catch.